### DISCUSSION TITLE
I contribute a open source project with 11PR successfully merge but in contribution list show only my 3 commits

### DISCUSSION THREAD

**Discussion by ashikjs** [community/community#68424] [New to GitHub]
A Open source project approved my 11commits but contribution list show only 3commites, when I check those commits. then I got those 8 committees are not verified because I use different email address reference when I was commiting from my local environment or device.

NO how can I solve it. I already added this email to my profile as new email address or alternative email but still it show only 3 commits .

**Comment by queenofcorgis** [comment]
Hi there, 

Have they shown up? They don't always show up right away and may take some time to hit your profile.
